mandatory/4-tax.js

```diff
@@ -17,7 +20,9 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(priceOfProduct) {
+  return "£".concat(calculateSalesTax(priceOfProduct).toFixed(2));
```

Comment:
There's a lot happening in this line, maybe separate it into 2 lines so it's easier to read. eg first line - call `calculateSalesTax`
second line - make the formatted string (you could use a string template?)